opt levels / debug should be under control of ports
disable -Werror as it often causes build issues on less-common archs

Index: auto/cc/test
--- auto/cc/test.orig
+++ auto/cc/test
@@ -71,7 +71,7 @@ case $NXT_CC_NAME in
 
         NXT_CFLAGS="$NXT_CFLAGS -std=gnu11"
 
-        NXT_CFLAGS="$NXT_CFLAGS -O"
+        #NXT_CFLAGS="$NXT_CFLAGS -O"
         NXT_CFLAGS="$NXT_CFLAGS -Wall -Wextra"
 
         #NXT_CFLAGS="$NXT_CFLAGS -Wunused-result"
@@ -115,7 +115,7 @@ case $NXT_CC_NAME in
 
         NXT_CFLAGS="$NXT_CFLAGS -std=gnu11"
 
-        NXT_CFLAGS="$NXT_CFLAGS -O"
+        #NXT_CFLAGS="$NXT_CFLAGS -O"
         NXT_CFLAGS="$NXT_CFLAGS -Wall -Wextra"
 
         #NXT_CFLAGS="$NXT_CFLAGS -Wunused-result"
